### ISSUE TITLE
jspm not respecting overwrite. ( jspm v 0.17.0-beta.29 )

### ISSUE THREAD

**Issue by DavidStrada** [jspm/jspm-cli#2142]
I have the following scenario, 

When installing Botstrap `4.0.0-alpha.5` it's not including the "main" entry point, so I wrote an overwrite to include it 

``` overwrite
jspm install npm:bootstrap@4.0.0-alpha.5 -o '{   "main": "dist/js/bootstrap" } '
```

It _should_ include it under `"npm:bootstrap@4.0.0-alpha.5"` but it's not.

package overwrites

``` json
//overwrites on package.json

"overrides": {
      "github:HubSpot/tether@1.3.7": {
        "main": "js/tether",
        "directories": {
          "lib": "dist"
        },
        "meta": {
          "js/tether.js": {
            "exports": "Tether",
            "format": "global"
          }
        }
      },
      "npm:bootstrap@4.0.0-alpha.5": {
        "jspmNodeConversion": false,
        "directories": {
          "lib": "."
        },
        "dependencies": {
          "jquery": "*",
          "tether": "github:HubSpot/tether@^1.1.1"
        },
        "meta": {
          "dist/js/bootstrap.js": {
            "deps": [
              "jquery",
              "tether"
            ],
            "exports": "$",
            "format": "global"
          }
        }
      },
      "npm:jquery@2.2.4": {
        "format": "amd"
      }
    }

```

**Comment by guybedford** [comment]
Yes this is a bug, I've added a fix in https://github.com/jspm/jspm-cli/commit/fd2990cb7ad589b8f2cbb63a5391eeb883c9cb42.

**Comment by DavidStrada** [comment]
@guybedford thanks! good to know that you already apply a fix for it.

**Comment by guybedford** [comment]
Released in 0.17.0-beta.32.
